01_fib/patrickmarabeas/main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// Fibonacci slice generator
```

Comment:
This is not the correct form for a doc string: https://blog.golang.org/godoc-documenting-go-code

Reply:
Hopefully this has been corrected...